packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -337,6 +337,7 @@ export default class ComboBox extends React.Component {
         inputValue={this.state.inputValue || ''}
         itemToString={itemToString}
         defaultSelectedItem={initialSelectedItem}
+        id={id}
```

Comment:
Do we know what the previous position was for the user-supplied ID? Was it on the button? It seems like downshift provides props for each id position (label, menu, input, id) and was curious which one matched the previous behavior